src/main/java/org/jabref/logic/importer/fileformat/MedlineImporter.java

```diff
@@ -55,6 +55,17 @@ public class MedlineImporter extends Importer implements Parser {
     private static final String KEYWORD_SEPARATOR = "; ";
 
     private static final Locale ENGLISH = Locale.ENGLISH;
+    private final XMLInputFactory xmlInputFactory;
+
+    public MedlineImporter() {
+        this.xmlInputFactory = XMLInputFactory.newInstance();
+        // prevent xxe (https://rules.sonarsource.com/java/RSPEC-2755)
+        xmlInputFactory.setProperty(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
+        // required for reading Unicode characters such as &#xf6;
+        xmlInputFactory.setProperty(XMLInputFactory.IS_COALESCING, true);
+        // TODO: decide if necessary, if disabled MedlineImporterTestNbib fails
+        xmlInputFactory.setProperty(XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, true);
+    }
```

Comment:
Why this change?

Reply:
I got no idea, I didn't modify this part.